packages/editor/src/components/post-featured-image/index.js

```diff
@@ -54,7 +57,7 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 							allowedTypes={ ALLOWED_MEDIA_TYPES }
 							modalClass="editor-post-featured-image__media-modal"
 							render={ ( { open } ) => (
-								<Button className="editor-post-featured-image__preview" onClick={ open } aria-label={ __( 'Edit or update the image' ) }>
+								<Button className="editor-post-featured-image__preview" onClick={ open } aria-label={ __( 'Edit or update the image' ) } aria-describedby={ !! mediaAltText ? 'Current image: ' + mediaAltText : 'The current image has no alternative text. The file name is: ' + path.basename( mediaSourceUrl ) }>
```

Comment:
Hi @Jackie6 thank you for your contribution.
Checking some documentation about aria-describedby https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute it seems this attribute should reference the id of an element that contains the description.
Hi guess @afercia suggestion also goes in the same direction:
> A possible solution is an aria-describedby attribute on the button to target a visually hidden description.

So I guess the solution is to have a hidden paragraph or span with the description and id and reference the id in this aria attribute.

Reply:
See also my comment here https://github.com/WordPress/gutenberg/pull/14752/files#r271206109:

> In other places in Gutenberg, we use [screen-reader-text](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/) class name if the message exists only for screen readers.

Reply:
@jorgefilipecosta @gziolo Thanks for you comments.